squids/core.py

```diff
@@ -45,10 +45,13 @@ class App:
         `boto3.session.Session.resource <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.resource>`_.
     """
 
-    def __init__(self, name: str, config: Optional[Dict] = None):
+    def __init__(
+        self, name: str, config: Optional[Dict] = None, serde: Type[Serde] = JSONSerde
```

Comment:
Rename config as a more explicit `boto3_config` or similar? still in alpha so should be fine to break, can be kind and offer a transition with a warning to.